explainaboard/utils/preprocessor.py

```diff
@@ -90,36 +90,31 @@ def process(self, s: str, resources: dict[str, Any]) -> str:
         language = resources['language']
 
         def remove_articles(text, lang):
-            if lang == 'en' or lang is None:
+            if lang in ['en', 'eng']:
```

Comment:
nit: I think relying on a library to normalize the language code is better.
For example: https://pypi.org/project/langcodes/